Library/Formula/docker-machine.rb

```diff
@@ -22,9 +22,13 @@ def install
     mkdir_p buildpath/"src/github.com/docker/"
     ln_sf buildpath, buildpath/"src/github.com/docker/machine"
 
-    system "make", "build"
-    bin.install Dir["bin/*"]
+    Dir.chdir(buildpath/"src/github.com/docker/machine/") do
+      # Hack to force ruby chdir into the symlinked path
+      ENV["PWD"] = buildpath/"src/github.com/docker/machine/"
+      system "make", "build"
+    end
```

Comment:
Not sure which upstream you mean I should submit a patch to - this is only a problem because the formular creates a symlink from `buildpath` to `buildpath/"src/github.com/docker/machine"` and then tries to build in there.

Ruby upstream are not likely to fix this, as it has been this way for ages.
`docker-machine` upstream are even less likely to fix it, as this is a problem we created in this formular by creating the symlink from within the extracted tar ball.

What we want to do is this (from the [docs](https://github.com/docker/machine/blob/master/CONTRIBUTING.md#local-go-15-development-environment)):
```bash
mkdir docker-machine
cd docker-machine
export GOPATH="$PWD"
go get github.com/docker/machine
cd src/github.com/docker/machine
```

The formular emulates this by creating a symlink `src/github.com/docker/machine` and cd'ing into it.

Is there a way to manually unpack the tar ball? That would let us create the directory `go get` creates, `cd` into it an build it just as `go get` does without the need for `symlink` shenanigans.